torch/_ops.py

```diff
@@ -1288,19 +1306,18 @@ class _OpNamespace(types.ModuleType):
         operation will already exist).
     """
 
-    def __init__(self, name):
+    __file__ = "torch.ops"
```

Comment:
typing here too

Reply:
@Skylion007 This is the one that `mypy` registered the base class having `Literal` type on. I'd argue that we shouldn't be adding that type to our code. If you would still prefer it, I will though.

Reply:
Oh because, it's a module type, yeah it makes sense it's a literal then... This mypy telling you should probably shouldn't override this lol